examples/src/main/java/com/adobe/platform/ecosystem/examples/constants/SDKConstants.java

```diff
@@ -25,6 +25,7 @@
     public static final String CONNECTION_IMS_ORG_KEY = "imsOrg";
     public static final String CONNECTION_HEADER_IMS_ORG_KEY = "x-gw-ims-org-id";
     public static final String CONNECTION_HEADER_JSON_CONTENT = "application/json";
+    public static final String CONNECTION_HEADER_JSON_FULL_CONTENT = "application/vnd.adobe.xed-full+json; version=1";
```

Comment:
Remove this constant as we consume version from `schemaRef` present inside DataSet model